formats/coming_soon/publisher/links.json

```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "replaced_content_item"
+  ],
+  "properties": {
+    "replaced_content_item": {
```

Comment:
After much discussion, we're going with `can_be_replaced_by` since that's a clearer description of what it's actually for.